src/experiments/logins/api.js

```diff
@@ -49,6 +49,20 @@ this.logins = class extends ExtensionAPI {
           //   - fooLogin = login as vanilla JS object
           //   - fooLoginInfo = login as nsILoginInfo
           //   - fooBag = (subset of) login fields as nsIPropertyBag
+          getLoginSavingEnabled(aHost) {
+            try {
+              return Services.logins.getLoginSavingEnabled(aHost);
```

Comment:
The trailing slash thing seems like a super annoying gotcha. I think we should either handle it here (strip it out or throw if it's detected), or document this annoyance in the schema docs.